.github/ISSUE_TEMPLATE/coverage_impact_report.md

```diff
@@ -0,0 +1,64 @@
+---
+name: Coverage Impact Report
```

Comment:
"Coverage Impact Report" -> "Improve Coverage Report"

Reply:
Done in 72eae4b.